site/content/docs/main/supported-providers.md

```diff
@@ -7,26 +7,31 @@ Velero supports a variety of storage providers for different backup and snapshot
 
 ## Velero supported providers
 
+{{< table caption="Velero supported providers" >}}
+
 | Provider                          | Object Store        | Volume Snapshotter           | Plugin Provider Repo                    | Setup Instructions            |
 |-----------------------------------|---------------------|------------------------------|-----------------------------------------|-------------------------------|
-| [Amazon Web Services (AWS)][7]    | AWS S3              | AWS EBS                      | [Velero plugin for AWS][8]              | [AWS Plugin Setup][35]        |
-| [Google Cloud Platform (GCP)][11] | Google Cloud Storage| Google Compute Engine Disks  | [Velero plugin for GCP][12]             | [GCP Plugin Setup][36]        |
-| [Microsoft Azure][9]              | Azure Blob Storage  | Azure Managed Disks          | [Velero plugin for Microsoft Azure][10] | [Azure Plugin Setup][37]      |
-| [VMware vSphere][39]              | 🚫                  | vSphere Volumes              | [VMware vSphere][39]                    | [vSphere Plugin Setup][40]    |
-| [Container Storage Interface (CSI)][44]| 🚫                 | CSI Volumes                  | [Velero plugin for CSI][44]             | [CSI Plugin Setup][45]        |
+| [Amazon Web Services (AWS)](https://aws.amazon.com)    | AWS S3              | AWS EBS                      | [Velero plugin for AWS](https://github.com/vmware-tanzu/velero-plugin-for-aws)              | [AWS Plugin Setup](https://github.com/vmware-tanzu/velero-plugin-for-aws#setup)        |
```

Comment:
`site/content/docs/v1.4/supported-providers.md` also needs to be updated

Reply:
thanks @ashish-amarnath! i thought i had updated that page already :)